lib/screens/authentication/import_key/import_words_screen.dart

```diff
@@ -24,7 +25,9 @@ class ImportWordsScreen extends StatelessWidget {
         builder: (context, state) {
           return Scaffold(
             bottomSheet: Padding(
-              padding: const EdgeInsets.all(16),
+              padding: Platform.isAndroid
+                  ? const EdgeInsets.all(16)
+                  : const EdgeInsets.only(bottom: 32, left: 16, right: 16),
```

Comment:
Why padding and not SafeArea?

Reply:
safeArea dont work for bottomSheet on IOS